functional-tests/README.md

```diff
@@ -0,0 +1,71 @@
+| Cluster       | Status    |
+| ------------- |-------------|
+| us-east-2 | [![Build Status](https://ci.centos.org/buildStatus/icon?job=devtools-che-functional-tests-periodical-openshift.io-2)](https://ci.centos.org/job/devtools-che-functional-tests-periodical-openshift.io-2) |
+| us-east-2a | [![Build Status](https://ci.centos.org/buildStatus/icon?job=devtools-che-functional-tests-periodical-prod-preview.openshift.io-2a)](https://ci.centos.org/view/Devtools/job/devtools-che-functional-tests-periodical-prod-preview.openshift.io-2a/) |
+| free-stg | [![Build Status](https://ci.centos.org/buildStatus/icon?job=devtools-che-functional-tests-periodical-prod-preview.openshift.io-free-stg)](https://ci.centos.org/view/Devtools/job/devtools-che-functional-tests-periodical-prod-preview.openshift.io-free-stg/) |
+
+# Functional tests for Eclipse Che on Openshift
+
+## Table Of Content
+
+* [What are these tests meant for](#what-are-these-tests-meant-for)
+* [How to run it](#how-to-run-it)
+* [Further details](#further-details)
+* [When are tests running](#when-are-tests-running)
+
+## What are these tests meant for
+
+These functional tests serve to verify that Eclipse Che on Openshift is working well. Tests include integration specific cases and some upstream tests to
+verify Che itself.
+
+There are some jobs running periodically on clusters us-east-2, us-east-2a and free-stg as you can see in the begining of this readme.
+
+## How to run it
```

Comment:
@Katka92 `How to run` only says about env var. Could you add some info about what should be done once env vars are set ? I assume it is `mvn clean install`. Also would be great to have a clarification how to run individual tests